index.js

```diff
@@ -38,8 +38,51 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const wrapArg = item => {
+	const alfredworkflow = {arg: item.arg, variables: item.variables};
+	const arg = JSON.stringify({alfredworkflow});
+	const newItem = Object.assign({}, item, {arg});
+	delete newItem.variables;
+	return newItem;
+};
+
+const formatMods = item => {
+	const copy = Object.assign({}, item);
+
+	Object.keys(item.mods).forEach(modKey => {
+		const modItem = item.mods[modKey];
+		copy.mods[modKey] = wrapArg(modItem);
+	});
+
+	return copy;
+};
+
+// see https://www.alfredforum.com/topic/9070-how-to-workflowenvironment-variables/
+// for documentation on setting environment variables from Alfred workflows
+const format = item => {
+	if (!isDefined(item)) {
```

Comment:
I would just throw if it's a non-object. Better to fail loudly than being silently "smart".

Could use: https://github.com/sindresorhus/is-plain-obj